defaults/main.yml

```diff
@@ -5,6 +5,7 @@ prometheus_volume_source: /srv/prometheus-data
 prometheus_volume_target: /prometheus-data
 prometheus_volume_mountinstruction: "{{prometheus_volume_source}}:{{ prometheus_volume_target }}"
 prometheus_container_name: prometheus
+prometheus_config_template: prometheus.yml.j2
```

Comment:
This is the same change as in #4 isn't it? Until I decide on what to do with this PR, the changes should be exclusives, so I'd revert 28e4258ce1d48936f9236a505ec302f400636f44 and f87398bc5ae858f5dee8ba7d28ee0207504698a8